src/main/java/nl/tudelft/ti2206/group9/ShaftEscape.java

```diff
@@ -0,0 +1,81 @@
+package nl.tudelft.ti2206.group9;
+
+import javafx.application.Application;
+import javafx.stage.Popup;
+import javafx.stage.Stage;
+import nl.tudelft.ti2206.group9.gui.AbstractScene;
+import nl.tudelft.ti2206.group9.gui.SplashScene;
+import nl.tudelft.ti2206.group9.util.GameObservable;
+import nl.tudelft.ti2206.group9.util.Logger;
+
+/**
+ * Starting point of the Application.
+ * Contains method to switch Scenes or show Popups.
+ * @author Maarten
+ */
+@SuppressWarnings("restriction")
+public class ShaftEscape extends Application {
+
+	private static Stage stage;
+	/** Width of the Window. */
+	public static final int WIDTH = 480 - 16;
+	/** Height of the Window. */
+	public static final int HEIGHT = 640;
+	/** Lock used so that the tickers won't use the Track concurrently. */
+	public static final Object TICKER_LOCK = new Object();
+
+	/**
+	 * Start the application in the SplashScene.
+	 * @param appStage the primary Stage for the Application. This is where
+	 * the scenes are shown in.
+	 */
+	@Override
+	public void start(Stage appStage) {
+		stage = appStage;
```

Comment:
*Auto-generated comment*

    Write to static field nl.tudelft.ti2206.group9.ShaftEscape.stage from instance method nl.tudelft.ti2206.group9.ShaftEscape.start(Stage)